docker/.env.dist

```diff
@@ -22,6 +22,7 @@ ALLOW_EXTERNAL_DEV_IPS=
 DEV_COOKIE=
 
 COMMENTS_FEATURE=inactive
+CREATE_ACCOUNT_FEATURE=inactive
```

Comment:
The `CREATE_ACCOUNT_FEATURE` env parameter needs to be added to `docker-compose.dev.yml` to pass it into the docker container.